scipy/stats/tests/test_distributions.py

```diff
@@ -4497,7 +4499,7 @@ def test_location_scale(
         # Test seems to be unstable (see gh-17839 for a bug report on Debian
         # i386), so skip it.
         if is_linux_32 and case == 'pdf':
-            pytest.skip("%s fit known to fail or deprecated" % dist)
+            pytest.skip("Test unstable on some platforms; see gh-17839, 17859")
```

Comment:
Oops, probably need to use `reason` keyword?
In any case, this message is based on the comment above it. Not sure where the original came from.